tasks/configs/copy.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  'create-artifacts-production': {
```

Comment:
Just a small nit but the full command would read better to me as `copy:dist-files-to-artifacts-dir`.